theme/favicon.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 199.7 184.2">
```

Comment:
We should remove these default favicons (png and svg). We have a favicon.ico under `public` that we should use.